pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -1695,6 +1695,22 @@ func (a *ApplicationRestoreController) cleanupRestore(restore *storkapi.Applicat
 			return fmt.Errorf("cancel restore: %s", err)
 		}
 	}
+	// Directly calling DeleteResourceExport with out checking backuplocation type.
+	// For other backuplocation type, expecting Notfound
+	crName := getResourceExportCRName(utils.PrefixRestore, string(restore.UID), restore.Namespace)
```

Comment:
We can just loop the same delete steps for both the CRs.

Reply:
Fixed